alpine/Dockerfile

```diff
@@ -12,23 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.6-alpine3.8
+FROM python:3.6-alpine3.10
 
 ENV PYTHONUNBUFFERED=1
 
-RUN apk add --update --no-cache openssl ca-certificates
-
+# Work in the /opt directory to persist the pip-compiled requirement list for
+# child images to use.
 WORKDIR /opt
 
-COPY wsgi-requirements.in ./
+RUN apk update
+
+RUN apk add --no-cache \
+    # Update OpenSSL and certificates
+    openssl \
+    ca-certificates \
+    # Libraries required for psycopg2
+    postgresql-dev \
+    python3-dev \
+    musl-dev
```

Comment:
If we want to keep these, it seems to me that they should be added to the `.buiild-deps` later on so that they will be removed again.

Reply:
Are you sure that even `postgresql-dev` isn't also needed at runtime?

Reply:
I'm getting this error at runtime, so I think the library needs to stay installed.

    ImportError: Error loading shared library libpq.so.5: No such file or directory (needed by /usr/local/lib/python3.6/site-packages/psycopg2/_psycopg.cpython-36m-x86_64-linux-gnu.so)

Reply:
See above.